appveyor.yml

```
@@ -100,6 +100,12 @@ artifacts:
 deploy_script:
  - ps: appveyor\scripts\deployScript.ps1
 
+deploy:
```

Note that this will now run the web hook for some more build types it didn't originally handle. E.g. pr and try builds. Will the web hook silently do nothing for these? 
There actually would be an advantage to have try builds handled and stored on our own file storage, as many times we do make try builds and appveyor's artifacts download daily limit policy hits people which was annoying. So, I'm good with this as long as the web hook won't fail on those build types.

---

The build processor is now set to handle all build types with details [in the documentation](https://github.com/nvaccess/flux-nvapps?tab=readme-ov-file#build-pipeline-and-deployment-process).  I've just added another change below to instruct AppVeyor not to skip PR build deploys which it does by default.  I have also put some logic in place to only retain 10 PR builds to keep it under control.

TL:DR, this PR in its current state should result in a build that will be accessible here:  
https://download.nvaccess.org/snapshots/pr/